Makefile

```diff
@@ -134,7 +134,9 @@ build/l10n-rel.html: assets/strings/locales.json isobork build/.d
 	awk 'BEGIN{while(getline<"isobork")i[$$1]=$$2} /:/{sub(/^[^"]*"/,""); sub(/".*/,""); borkd=i[$$0]; if(!borkd)borkd=$$0; print "<link rel=\"alternate\" hreflang=\""borkd"\" href=\"index."$$0".html\">"}' $^ > $@
 
 build/index.%.html: build/strings/%.json build/index.localiseme.html $(CONFIG) tools/read-conf.py tools/localise-html.py
-	./tools/localise-html.py -c $(CONFIG) build/index.localiseme.html $< $@
+	./tools/localise-html.py -c $(CONFIG) build/index.localiseme.html $< $@.tmp
```

Comment:
Seems possible to just pipe into `htmlmin`.

Reply:
The script expects an output path. It doesn't print to `stdout`.

Reply:
https://htmlmin.readthedocs.io/en/latest/quickstart.html#command-line

Reply:
I was referring to `localise-html.py` not printing to `stdout`.
I am well aware that I could pipe that into `htmlmin` if it did.

Reply:
Change it then?